File: app/src/main/java/com/lumiyaviewer/lumiya/dao/GroupMemberList.java
package com.lumiyaviewer.lumiya.dao;

import java.util.UUID;

public class GroupMemberList {
    private UUID groupID;
    private UUID requestID;

    public GroupMemberList() {
    }

    public GroupMemberList(UUID uuid) {
        this.groupID = uuid;
    }

    public GroupMemberList(UUID uuid, UUID uuid2) {
        this.groupID = uuid;
        this.requestID = uuid2;
    }

    public UUID getGroupID() {
        return this.groupID;
    }

    public UUID getRequestID() {
        return this.requestID;
    }

    public void setGroupID(UUID uuid) {
        this.groupID = uuid;
    }

    public void setRequestID(UUID uuid) {
        this.requestID = uuid;
    }
}

File: app/src/main/java/com/lumiyaviewer/lumiya/dao/GroupMemberList.kt
package com.lumiyaviewer.lumiya.dao

import java.util.UUID

class GroupMemberList {
    private var groupID: UUID? = null
    private var requestID: UUID? = null

    constructor()

    constructor(uuid: UUID?) {
        this.groupID = uuid
    }

    constructor(uuid: UUID?, requestID: UUID?) {
        this.groupID = uuid
        this.requestID = requestID
    }

    fun getGroupID(): UUID? = groupID

    fun getRequestID(): UUID? = requestID

    fun setGroupID(uuid: UUID?) {
        this.groupID = uuid
    }

    fun setRequestID(uuid: UUID?) {
        this.requestID = uuid
    }
}
